packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1285,26 +1294,48 @@ def generate_unique_func_name(context: TransformContext) -> TransformContext:
     return context
 
 
-def process_code(
-    context: TransformContext,
+def _check_global(code_tree: ast.Module) -> GlobalsVisitor | SyftWarning:
```

Comment:
we can remove `_` from function name, possibly rename to `check_for_global_vars` ?